src-tauri/src/gui/commands/utils_commands.rs

```diff
@@ -13,7 +13,7 @@ use tauri::AppHandle;
 use num_cpus;
 use anyhow::{Result};
 
-use crate::gui;
+use crate::gui::{self, app_state};
```

Comment:
You added app_state to the import list here, but it is never used. Introducing unused imports is unnecessary noise and should be caught before the PR is submitted.

It would be extremely helpful if you reviewed the GitHub diff yourself before requesting a review. Basic self-review should catch obvious issues like unused imports, leftover code, or incomplete refactoring. This is a matter of professional courtesy and respect for the reviewer’s time.

Please ensure you perform a thorough pass over your changes before assigning the PR for review.